src/coreComponents/constitutive/fluid/multifluid/compositional/models/EquationOfState.hpp

```diff
@@ -37,12 +37,14 @@ namespace compositional
 enum class EquationOfStateType : integer
 {
   PengRobinson,
-  SoaveRedlichKwong
+  SoaveRedlichKwong,
+  ImmiscibleWater
```

Comment:
i am probably missing something
a bit confused here - how will it know what eos is to use for hc?

Reply:
This is specified in the xml. The phases are "lined up" with the EOS e.g.
```xml
 <CompositionalThreePhaseFluidLohrenzBrayClark
      name="blackOil"
      phaseNames="{ oil, gas, water }"
      equationsOfState="{ pr, pr, immiscibleWater }" />
```
However, I think since the `immiscibleWater` "EOS" is implied in the element name (`CompositionalThreePhaseFluidLohrenzBrayClark`), there is no need to specify this here. I'm inclined to remove it.